checks/system/integration/daint.py

```diff
@@ -175,7 +175,7 @@ def create_checks(check):
 
     check('grep -q "/capstor/scratch/cscs /capstor/scratch/cscs lustre"     /proc/mounts || echo FAILED', not_expected=r'FAILED')
     check('grep -q "/capstor/store/cscs /capstor/store/cscs lustre"         /proc/mounts || echo FAILED', not_expected=r'FAILED')
-    check('grep -q "/capstor/users/cscs /users.OLD lustre"                  /proc/mounts || echo FAILED', not_expected=r'FAILED')
+    check('grep -q "/capstor/users/cscs /users.OLD lustre"                  /proc/mounts || echo FAILED', expected=r'FAILED')
```

Comment:
```suggestion
    check('grep -q "/capstor/users/cscs /users.OLD lustre"                  /proc/mounts || echo FAILED', not_expected=r'FAILED')
```
I think we should fix like this, not ignore the FAILED message

Reply:
It's not there anymore, so in this way I make sure that it's not mounted.